README.md

```diff
@@ -57,7 +57,7 @@ implementation "org.jetbrains.kotlin:kotlin-stdlib:1.3.31"
 The main step required in order to use the Access Checkout SDK is the initialisation of an instance of `AccessCheckoutClient`. 
 Interaction with this class will allow developers to obtain a session state from Access Worldpay services. 
 
-This can be achieved by invoking the `init()` method of this class and by providing the required dependencies, more details provided in the following sections.
+This can be achieved by making use of the builder method for this class and by providing the required dependencies, more details provided in the following sections.
```

Comment:
I would leave the README out of this PR and include this in the other documentation ticket which is more client facing.